### DISCUSSION TITLE
TTL to USB

### DISCUSSION THREAD

**Discussion by rkidwell1** [mr-manuel/venus-os_dbus-serialbattery#147] [General]
Hello,

I am going to be connecting my 280ah 12v battery bank with a JK BMS to my CERBO GX for monitoring only (the CERBO GX is also connected to my 560ah 48v battery bank using a REC BMS). 

I’m slightly confused about the TTL to USB converter that I need. The directions show to use an isolated TTL to USB and a 12v to 5v buck converter to power the converter off of the 12v power output on the GPS plug on the JK.  

There is a YouTube video by @Jasonschmidtschannel that shows a very simple TTL to USB converter/cable where he does not have the power hooked up (just TX, RX, and Ground).

[https://youtu.be/ZwTxyB65Aas?si=ANnG9Vg24jI2ZCmz](url)

![IMG_5977](https://github.com/user-attachments/assets/c419d61a-6b90-407a-b5ed-0e6a083836ae)
![IMG_5976](https://github.com/user-attachments/assets/7215eba9-4920-485e-aad4-ad96ae5fe52c)

Is any reason this cable would not work in the same manner with just TX, RX, and Ground connected to the GPS connector (using a JST MX 1.25) on the BMS? 

https://a.co/d/7oxHVzg

Thank you!

**Comment by mr-manuel** [comment]
The adapter is not isolated. Check the docs what that means.

**Comment by rkidwell1** [comment]
Okay, this one is isolated. Do I need to run a 12v to 5v buck converter from the battery voltage to VCC pin and switch the jumper to 5v? Or do I need a 3.3v buck converter?

https://a.co/d/2yNlqpm

https://www.deshide.com/product-details.html?pid=303386&_t=1661499873

![IMG_5978](https://github.com/user-attachments/assets/07e5b42c-7c03-4588-8ec7-0212796bb6fe)

**Reply by mr-manuel** [reply]
Does not matter. Important is that you set the jumper to the voltage you chose.
